.github/workflows/test.yml

```diff
@@ -10,32 +10,48 @@ jobs:
   short:
     runs-on: macos-latest
     steps:
-        - uses: actions/checkout@v4
-        - uses: actions/setup-go@v5
-          with:
-            go-version-file: go.mod
-        - run: make test
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - run: make test
 
   coverage:
     runs-on: ubuntu-latest
     steps:
-        - uses: actions/checkout@v4
-        - uses: actions/setup-go@v5
-          with:
-            go-version-file: go.mod
-        - run: make cover.out
-        - uses: vladopajic/go-test-coverage@v2
-          with:
-            config: .testcoverage.yml
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - run: make cover.out
+      - uses: vladopajic/go-test-coverage@v2
+        with:
+          config: .testcoverage.yml
 
   e2e:
     runs-on: ubuntu-latest
     steps:
-        - uses: actions/checkout@v4
-          with:
-            submodules: 'recursive'
-        - uses: actions/setup-go@v5
-          with:
-            go-version-file: go.mod
-        - uses: foundry-rs/foundry-toolchain@v1
-        - run: make e2e
+      - uses: actions/checkout@v4
+        with:
+          submodules: "recursive"
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - uses: foundry-rs/foundry-toolchain@v1
+      - run: make e2e
+
+  bindings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: "recursive"
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - name: Install abigen
+        run: go install github.com/ethereum/go-ethereum/cmd/abigen@latest # pin?
```

Comment:
Can we have an `install-abigen` target in the makefile for this?

Reply:
Added here: eb00aa9.

I think I'm happy enough for these to stay `latest` - having it in CI will enforce consistency between machines.